fastplotlib/graphics/linecollection.py

```diff
@@ -2,10 +2,9 @@
 import pygfx
 from typing import Union, List
 
-from .line import LineGraphic
+from fastplotlib.graphics.line import LineGraphic
 from typing import *
-from ._base import Interaction
-
+from fastplotlib.graphics._base import Interaction
```

Comment:
same as above, always use relative imports from within a package, I'm guessing this was pycharm's fault